tests/test_bookmarks.py

```diff
@@ -0,0 +1,207 @@
+import copy
+from datetime import datetime, timedelta
+from dateutil.parser import parse
+from base import SnapchatBase
+from tap_tester import runner, connections, menagerie
+
+class SnapchatBookmarksTest(SnapchatBase):
+
+    parent_child_mapping = {
+        # parent: child
```

Comment:
isn't comment is like **child: parent**, because in all your keys are child stream with respect to its value stream

Reply:
Right, updated the comment.